components/phpunit_bridge.rst

```diff
@@ -0,0 +1,114 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
```

Comment:
Well, this is a bridge and not a component. But we don't have a bridges section on the docs though, so the components section is the best place to put it now.